htlcswitch/link.go

```diff
@@ -2404,6 +2438,64 @@ func (l *channelLink) handleUpstreamMsg(msg lnwire.Message) {
 		// Update the mailbox's feerate as well.
 		l.mailBox.SetFeeRate(fee)
 
+	case *lnwire.Stfu:
+		handleErr := func(err error) {
```

Comment:
Could fold this into a `handleStfu` method mainly to reduce some of the vertical distance here, and also allow just that resp handling to be dested w/o needing to spin up the entire state machine.

Reply:
Great suggestion.

Reply:
Looks to still be relevant.